app/javascript/components/organisms/StatusModalWrapper.js

```diff
@@ -66,7 +67,7 @@ class StatusModalWrapper extends React.Component {
                   wrapperClasses={['subStatus']} />
               </React.Fragment>
             )}
-            <label className='sr-only' htmlFor='status-comment' id='status-comment-label'>Comment</label>
+            <label className={`form-label ${formUtils.submitErrors(formApi).comment ? 'error' : ''}`} id='status-comment-label'>Comment{statusRequiresComments(status, subStatus) ? ' (required)' : ''}</label>
```

Comment:
is it feasible to update this to add a [blocknote](https://github.com/Exygy/sf-dahlia-lap/blob/116d1351172dc1057acc68f2989a22c92615ae58/app/javascript/utils/form/final_form/Field.js#L10) or does that not make sense if we're not using react-final-form yet?